src/main/java/org/tikv/common/region/AbstractRegionStoreClient.java

```diff
@@ -230,6 +231,7 @@ private Boolean seekLeaderStore(BackOffer backOffer) {
             // switch to leader store
             store = currentLeaderStore;
             updateClientStub();
+            return true;
```

Comment:
you fix a bug here?

Reply:
Yes. If it returns false here, the region info will update for nothing.

Reply:
Is there a testcase to verify this fix?

Reply:
Yes. `SeekLeaderStore.testSeekLeader` cover this case. If return false, the tests will fail.

Reply:
should it be cherry-picked to release-3.1?